_posts/2020-05-04-this-week-in-glean.md

```diff
@@ -0,0 +1,266 @@
+---
+permalink: "/{{ year }}/{{ month }}/{{ day }}/this-week-in-glean"
+title: "This Week in Glean: Bytes in Memory (on Android)"
+published_date: "2020-05-04 12:00:00 +0200"
+layout: post.liquid
+data:
+  route: blog
+  tags:
+    - mozilla
+    - rust
+---
+
+(“This Week in Glean” is a series of blog posts that the Glean Team at Mozilla is using to try to communicate better about our work. They could be release notes, documentation, hopes, dreams, or whatever: so long as it is inspired by Glean.)
+
+Last week's blog post: [ This Week in Glean: Glean for Python on Windows](https://blog.mozilla.org/data/2020/04/27/this-week-in-glean-glean-for-python-on-windows/)
+by Mike Droettboom.
+All "This Week in Glean" blog posts are available on the [Mozilla Data blog](https://blog.mozilla.org/data/category/glean/).
+
+---
+
+With the Glean SDK we follow in the footsteps of [other teams](https://github.com/mozilla/application-services) to build a cross-platform library to be used in mobile applications.
+In this blog post we're going to take a look at how we transport some rich data across the FFI boundary to be reused on the Kotlin side of things.
```

Comment:
Can you provide a bit more context about what the article is about? By reading this I'd expect the article to talk about FFI stuff , which is something you mostly worked on. However, the article also describes bits of the upload mechanism, which is what most of the implementation is about. I'd spend a few more words on this. If this is about describing the upload mechanism as well, then I'd provide a super tl;dr about how it should work.

Reply:
ok with linking [to the docs?](https://github.com/mozilla/glean/blob/8e0599b4df6c1a08f985e4ad5328fcf81a56a084/docs/dev/core/internal/upload.md)

this is really more about the approach than about uploading.

Reply:
> 
> 
> ok with linking [to the docs?](https://github.com/mozilla/glean/blob/8e0599b4df6c1a08f985e4ad5328fcf81a56a084/docs/dev/core/internal/upload.md)
Yes !

> this is really more about the approach than about uploading.

Perfect! Even just static exactly this somewhere in this paragraph would make the rest more coherent, I believe.